spark/src/test/scala/org/apache/comet/parquet/ParquetReadSuite.scala

```diff
@@ -1217,34 +1217,6 @@ abstract class ParquetReadSuite extends CometTestBase {
     }
   }
 
-  test("schema evolution") {
```

Comment:
Can we update the test rather than remove it?

Reply:
Seems to me this only tests `CometConf.COMET_SCHEMA_EVOLUTION_ENABLED`. Since I removed the config, I think this test is not needed any more.
I currently did my test in Iceberg. I am thinking maybe we can add an iceberg-integration module to have the iceberg related test, or depend on iceberg CI (https://github.com/apache/datafusion-comet/pull/1715)?